### ISSUE TITLE
Why loadavg is 10 minutes

### ISSUE THREAD

**Issue by opensourcegeek** [danburkert/procinfo-rs#33]
Just checking why you've named load average struct with 10 min, as it looks like 1, 5 and 15 mins values according to `man proc`. Are there any other post processing happening to convert it into 10 mins - though I couldn't see when skimming through source code?

**Comment by danburkert** [comment]
Yep, this certainly seems like a mistake.  Thanks for the report, any interest in submitting a PR?
